app/src/main/contrib/actionTypes/RunParametersTypes.ts

```diff
@@ -0,0 +1,43 @@
+import {ModelRunParameterSet} from "../../shared/models/Generated";
+
+export interface tokenData {
+    id: number,
+    token: string
+}
+
+export enum RunParametersTypes {
+    RUN_PARAMETERS_SETS_FETCHED = "RUN_PARAMETERS_SETS_FETCHED",
+    RUN_PARAMETERS_TOKEN_FETCHED = "RUN_PARAMETERS_TOKEN_FETCHED",
+    RUN_PARAMETERS_SET_UPLOAD_STATUS = "RUN_PARAMETERS_SET_UPLOAD_STATUS",
+}
+
+export interface RunParametersSetsFetched {
+    type: RunParametersTypes.RUN_PARAMETERS_SETS_FETCHED;
+    data: ModelRunParameterSet[];
+}
+
+export interface RunParametersTokenFetched {
+    type: RunParametersTypes.RUN_PARAMETERS_TOKEN_FETCHED;
+    data: tokenData;
+}
+
+export enum RunParametersUploadStatus {
+    off = 'off' ,
```

Comment:
Could you explain how these statuses are used? Do we need 3 statuses or can we just have a boolean value that is set to true when the upload is complete?